### DISCUSSION TITLE
Broken Link on 'Setting Up Meta" Documentation page.

### DISCUSSION THREAD

**Discussion by AgedLace** [mkdocs-publisher/mkdocs-publisher#28] [General]
Under 'Navigation automatic generation'

In the sentence "Probably the easiest way of file ordering is to provide some prefix with the digits (take a look at [this documentation repository ](https://github.com/mkusz/mkdocs-publisher/tree/main/documentation)for better understanding)."

this link no longer exists due to changing the 'documentation' directory.

**Comment by mkusz** [comment]
I will update that with next commit to the repo. Thx for catching it.

**Comment by AgedLace** [comment]
I have created another test repo - having issues with wikilinks, but I haven't had time to fully test it yet.  Haven't tested direct links yet.

**Reply by mkusz** [reply]
Remember to set up relative links in Obsidian (options -> Files & Links -> New link format = Relative path to file] and when you are in the teaser section of a blog post, wiki links from that section do not work correctly when you are seeing post in blog (when you see a full post, links works correctly).

**Comment by AgedLace** [comment]
I'm hoping to get to this over the weekend.  I have to make longer test posts as I'm not getting the <more> teaser section.  Also noticed that the Blog post titles don't 'link' either.  Will be checking that out, too.

**Reply by mkusz** [reply]
If you will need any help, just let me know.
